files/vacuum_full_tables.sh

```diff
@@ -1,39 +1,43 @@
 #!/bin/bash
 
-if [ "$1" = "" ]; then
-  echo "Usage: $0 <(facts, catalogs, or other) tables to VACUUM FULL> "
-  exit
-fi
+usage() {
+  cat <<EOF
+echo "usage: $0 <table>"
+where <table> is one of: facts catalogs other
+EOF
+  exit 1
+}
+# Print usage if given 0 arguments
+(( $# == 0 )) && usage
 
```

Comment:
Could both scripts implement the same style of usage function and while/case param parsing?